### PR TITLE
feat(ingestion) Add more info to glue entities

### PR DESCRIPTION
**Pull request by skrydal**
This PR adds subTypes=["Table"] to glue datasets (tables) and also ingest Description of glue database (similarly to Description of the tables).

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by hsheth2 — APPROVED** [review]
I made a minor tweak around the mcp generation logic. Overall LGTM.

One thing before we can merge this: I need to check if the subType piece is case-sensitive - we have used `table` elsewhere in the codebase but `Table` here, so I want to verify that it won't cause any issues.

**Comment by skrydal** [comment]
@hsheth2 thank you for the comment, I am afraid introduced changes are breaking the tests, there are fails which weren't there previously: https://github.com/datahub-project/datahub/actions/runs/3049282407/jobs/4915193424 - I would fix it but I am not sure I understand why they were introduced.

**Comment by hsheth2** [comment]
That's my bad - fixed up the tests

**Pull request merged**
